backend/src/main/java/com/empire/EntryServlet.java

```diff
@@ -259,6 +259,14 @@ private String getAdvancePoll() {
 						for (String mail : emails.keySet()) {
 							mail(mail, "👑 Empire: Turn Advances", emails.get(mail).replace("%GAMEID%", "" + gameId));
 						}
+						if (w.gameover) {
+							ActiveGames newActiveGames = new ActiveGames();
+							Entity activeGames = new Entity("ACTIVEGAMES", "_");
```

Comment:
Not: move the Entity instantiation down a few lines to where it starts getting used.